### ISSUE TITLE
File resets after certain file size

### ISSUE THREAD

**Issue by feelwood** [Phobos97/clan-activity-tracker#1]
After using the plugin for about a week, it seems to have only tracked 6 members out of ~490.
I'm unsure if this is maybe a conflict with other plugins, but I do also use Clan Member List Sorting and Copy FC/CC users to clipboard.
I also multi-client with the same plugin configurations. Not sure if that matters.

**Comment by feelwood** [comment]
After further investigation, it's 100% the multi-clienting. I guess because they're both trying to change the file at the same time, perhaps. Both accounts are in the same clan.

**Comment by Phobos97** [comment]
I was not really able to reproduce some of your bugs but ive added an option to use the rsn of the account you are logged in on as part of the filename, this will prevent any multi logging bugs (it was also double counting messages before). 

Note that it only adds online people to the log at the moment. If you are still missing people in the log that are currently online you can try leaving and re-entering the fc/cc (yes you can leave a clan chat without fully leaving the clan).

I have just pushed this update to the Plugin Hub but it might be a day or so before the changes are added in the client.

**Comment by feelwood** [comment]
Seems like I'm still getting the issue after the update. After a while the file just resets and starts from scratch. I'm not entirely sure what's causing it. I've been backing up the file and saving it elsewhere to keep in case of resets but yeah, for some reason, the file hits about 8kb and then just resets.

**Comment by feelwood** [comment]
I've done a little more testing. I've backed up a file right before it resets. Any more data added to this file seems to reset it. So any chat messages, people logging in etc will reset the file back to a fresh file.
It's 9kb in size and for some reason has a random N at the bottom of the list.

[Hallowed_RS_Touch Wood_Log1 - Before Error Log.csv](https://github.com/Phobos97/clan-activity-tracker/files/8475783/Hallowed_RS_Touch.Wood_Log1.-.Before.Error.Log.csv)

I've attached the file in case it helps at all but looks to me like after a certain file size the plugin seems to have issues writing more data or something along those lines.
